carma_cooperative_perception/src/msg_conversion.cpp

```diff
@@ -281,6 +300,14 @@ auto to_detection_list_msg(
     quat_tf.setRPY(0, 0, remove_units(units::angle::radian_t{enu_yaw}));
     detection.pose.pose.orientation = tf2::toMsg(quat_tf);
 
+    try {
+      // Pose covariance is x, y, z, roll, pitch, yaw
```

Comment:
Might be helpful to explicitly mention at these lines that we are dealing with flattened 6x6 matrix

Reply:
Updated